VoiceOfAmerica/ViewControllers/VoteViewController.swift

```diff
@@ -92,6 +123,29 @@ extension VoteViewController {
             }
         }
     }
+}
+
+extension VoteViewController: UISearchBarDelegate {
+    func searchBarCancelButtonClicked(_ searchBar: UISearchBar) {
+        self.configureSearchBar()
```

Comment:
Do we reconfigure every cancel so that white bar doesn't appear when we dismiss?

Reply:
Yeah. I didn't do too much research into this but totally open to suggestions.

Reply:
@meieri thoughts on landing as-is for the time being? This work-around is computationally inexpensive and looks good. Let's discuss what we want this behavior to look like - and somehow hiring a designer. (Do we know any design majors? Lol)